public_suffix_list.dat

```diff
@@ -12404,6 +12404,10 @@ static-access.net
 // Submitted by Ales Krajnik <ales.krajnik@craynic.com>
 realm.cz
 
+// Crisp IM SAS : https://crisp.chat/
+// Submitted by Baptiste Jamin <baptiste@crisp.chat>
```

Comment:
If you can, make this a role based email so it isn't tied to a particular employee.

Reply:
Done.